charts/datagovuk/templates/opensearch-sts/pvc.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.opensearch.persistence.enabled }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: {{ .Values.opensearch.persistence.persistentVolumeClaimName }}
+spec:
+  resources:
+    requests:
+      storage: 1Gi
```

Comment:
This seems kinda small but I don't know how big the dataset is. Just as long as you have plenty of headroom.

Reply:
I left it running over the weekend, only used 22% of capacity so I reckon its ok